supertokens_python/recipe/emailverification/ev_claim.py

```diff
@@ -0,0 +1,82 @@
+# Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
+#
+# This software is licensed under the Apache License, Version 2.0 (the
+# "License") as published by the Apache Software Foundation.
+#
+# You may not use this file except in compliance with the License. You may
+# obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+from typing import Dict, Any
+
+from supertokens_python.recipe.session.claim_base_classes.boolean_claim import (
+    BooleanClaim,
+    BooleanClaimValidators,
+)
+from supertokens_python.recipe.session.interfaces import (
+    SessionClaimValidator,
+    JSONObject,
+    ClaimValidationResult,
+)
+from supertokens_python.types import MaybeAwaitable
+from supertokens_python.utils import get_timestamp_ms
+
+
+class IsVerifiedSCV(SessionClaimValidator):
+    def __init__(
+        self,
+        claim: BooleanClaim,
+        has_value_validator: SessionClaimValidator,
+        refetch_time_on_false_in_seconds: int,
+        max_age_in_seconds: int,
+    ):
+        super().__init__("st-ev-is-verified")
+        self.claim: BooleanClaim = claim  # TODO: Should work without specifying type of self.claim (no pyright errors)
+        self.has_value_validator = has_value_validator
```

Comment:
this is not needed either cause you can get it from the validator class

Reply:
done